### DISCUSSION TITLE
Should xmake print every file on extracting tarballs if `-v` is set?

### DISCUSSION THREAD

**Discussion by xq114** [xmake-io/xmake#2711] [Ideas]
目前xmake只要声明了-v，在解压tar.gz文件的时候tar程序就会加入-v选项，根据文档 https://www.gnu.org/software/tar/manual/html_section/verbose.html 这一选项作用仅仅是输出压缩包中的所有文件和目录。这一输出对于问题调试完全没有帮助，但却大大增加了输出的文字量，导致真正出问题的地方难以定位甚至无法查看到（见 https://github.com/xmake-io/xmake-repo/runs/7965937148?check_suite_focus=true ）。而使用7z进行解压就没有这个问题，因为7z没有加verbose的选项。

https://github.com/xmake-io/xmake/blob/master/xmake/modules/utils/archive/extract.lua

是否可以在这里统一一下解压过程，不要给tar设置-v选项？

**Comment by waruqi** [comment]
直接来个 pr 把
